packages/models/src/currencies.model.ts

```diff
@@ -1,7 +1,7 @@
 import { LiteralUnion } from './types.utils';
 
-export type CryptoCurrencies = LiteralUnion<'BTC' | 'STX', string>;
+export type CryptoCurrency = LiteralUnion<'BTC' | 'STX', string>;
```

Comment:
Maybe we can call this `Token` instead of `CryptoCurrency` if we are changing it anyway?

Reply:
I'm not sure `Token` is the right choice here considering an NFT is a `token`? The naming here is meant to identify those that are native gas tokens, or `cryptocurrencies`:
https://www.gemini.com/cryptopedia/cryptocurrencies-vs-tokens-difference

Reply:
OK 👍 I just see us labelling them as `tokens` in the mobile designs now, we have `tokens` + `collectibles`